ql.mm

```diff
@@ -371,463 +384,421 @@
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
-  $( Identity law. $)
+  $( Identity law.  (Contributed by NM, 9-Aug-1997.) $)
   id $p |- a = a $=
     ( wn ax-a1 ax-r1 ax-r2 ) AABBZAACZAFGDE $.
-    $( [9-Aug-97] $)
 
-  $( Justification of definition ~df-t of true ( ` 1 ` ).  This shows that the
-     definition is independent of the variable used to define it. $)
+  $( Justification of definition ~ df-t of true ( ` 1 ` ).  This shows that the
+     definition is independent of the variable used to define it.  (Contributed
+     by NM, 9-Aug-1997.) $)
   tt $p |- ( a v a ' ) = ( b v b ' ) $=
     ( wn wo ax-a4 ax-r1 ax-a2 ax-r2 ) AACDZIBBCDZDZJIJIDZKLIJAEFJIGHIBEH $.
-    $( [9-Aug-97] $)
 
   ${
     cm.1 $e |- a = b $.
-    $( Commutative inference rule for ortholattices. $)
+    $( Commutative inference rule for ortholattices.  (Contributed by NM,
+       26-May-2008.)  (Revised by NM, 26-May-2008.) $)
```

Comment:
Is the revision tag needed/useful? It is the same day...

Reply:
It was autogenerated from the date markings. I can remove it.